contract/models/contract.py

```diff
@@ -614,14 +614,14 @@ def _add_contract_origin(self, invoices):
                 move.message_post(
                     body=(
                         _(
-                            (
-                                "%(msg)s by contract <a href=# data-oe-model=contract.contract"
-                                " data-oe-id=%(contract_id)d>%(contract)s</a>."
-                            ),
-                            msg=move._creation_message(),
```

Comment:
AFAIK, this one is correct.

Reply:
I don't think so. It's not in .pot file

Reply:
Uhm, but you didn't update POT files with this PR to check if it's working.

Reply:
Yes, the change is included in a regenerated .pot file.
Do we update .pot/.po file by PR's?
Or only by the GH action and weblate?